internal/datasource/metric/provider.go

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package metric
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/nginx/agent/v3/internal/config"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+)
+
+const (
+	// tenantID is static for now.
+	tenantID         = "7332d596-d2e6-4d1e-9e75-70f91ef9bd0e"
+	serviceNamespace = "nginx"
+	readInterval     = 10 * time.Second
```

Comment:
can we add this to the centralized configuration?

Reply:
This is already added to the config, this `const` is actually unused. The code below also already uses the value from the config 😄 

Removed the `const`!